docs/tutorials/typescript/work-queue-slack-app/index.mdx

```diff
@@ -0,0 +1,991 @@
+---
+id: index
+title: Build a Work Queue Slackbot with Temporal in Typescript
+description: Learn how to build a Slash Command Slack App using Temporal to manage ork queues without a traditional database.
+sidebar_label: Build a Work Queue Slackbot with Temporal in Typescript
+tags:
+  - typescript
+  - slack-app
+keywords:
+  - Slash command Slack App tutorial
+  - Temporal TypeScript application
+  - Slack App development
+  - Temporal workflows
+  - Persist Slack app data
+  - No database Slack app
+  - Temporal signals and queries
+  - Work queue management in Slack
+  - Temporal Client setup
+  - Interactive Slack commands
+  - Temporal TypeScript SDK
+  - Deploy Slack app production
+  - Manage work queues without database
+  - Temporal Cloud integration
+  - Slack bot development
+---
+
+### Introduction
+
+When you're creating a Slash command Slack App, you might need to persist data between interactions.
```

Comment:
Probably need one sentence before this with a link somewhere explaining what a "Slash command" Slack App actually is